docs/sphinx/advanced_configuration.rst

```diff
@@ -14,24 +14,25 @@ argument is a boolean option, and  can be turned on or off:
 
-      ===========================  ======== ===============================================================================
-      Variable                     Default  Meaning
-      ===========================  ======== ===============================================================================
-      ``ENABLE_CUDA``              Off      Enable CUDA support
-      ``ENABLE_HIP``               Off      Enable HIP support
-      ``ENABLE_NUMA``              Off      Enable NUMA support
-      ``ENABLE_STATISTICS``        Off      Enable collection of memory statistics
-      ``ENABLE_TESTING``           On       Build test executables
-      ``ENABLE_BENCHMARKS``        On       Build benchmark programs
-      ``ENABLE_LOGGING``           On       Enable Logging within Umpire
-      ``ENABLE_SLIC``              Off      Enable SLIC logging
-      ``ENABLE_BACKTRACE``         Off      Enable backtraces for allocations
-      ``ENABLE_BACKTRACE_SYMBOLS`` Off      Enable symbol lookup for backtraces
-      ``ENABLE_TOOLS``             Off      Enable tools like replay
-      ``ENABLE_DOCS``              Off      Build documentation (requires Sphinx and/or Doxygen)
-      ``ENABLE_C``                 Off      Build the C API
-      ``ENABLE_FORTRAN``           Off      Build the Fortran API
-      ===========================  ======== ===============================================================================
+    ============================  ======== ===========================================================================
+    Variable                      Default  Meaning
+    ============================  ======== ===========================================================================
+    ``ENABLE_CUDA``               Off      Enable CUDA support
+    ``ENABLE_HIP``                Off      Enable HIP support
+    ``ENABLE_NUMA``               Off      Enable NUMA support
+    ``ENABLE_FILE_RESOURCE``      Off      Enable FILE support      
+    ``ENABLE_STATISTICS``         Off      Enable collection of memory statistics
+    ``ENABLE_TESTING``            On       Build test executables
+    ``ENABLE_BENCHMARKS``         On       Build benchmark programs
+    ``ENABLE_LOGGING``            On       Enable Logging within Umpire
+    ``ENABLE_SLIC``               Off      Enable SLIC logging
+    ``ENABLE_BACKTRACE``          Off      Enable backtraces for allocations
+    ``ENABLE_BACKTRACE_SYMBOLS``  Off      Enable symbol lookup for backtraces
+    ``ENABLE_TOOLS``              Off      Enable tools like replay
+    ``ENABLE_DOCS``               Off      Build documentation (requires Sphinx and/or Doxygen)
+    ``ENABLE_C``                  Off      Build the C API
+    ``ENABLE_FORTRAN``            Off      Build the Fortran API
+    ============================  ======== ===========================================================================
```

Comment:
@davidbeckingsale So we did have a table that I could've used as reference but you were missing a "=" causing the table to break. That's why you couldn't see it